yarn.lock

```diff
@@ -254,7 +254,7 @@
   dependencies:
     "@types/chai" "*"
 
-"@types/chai@*":
+"@types/chai@*", "@types/chai@^4.2.7":
```

Comment:
Is this needed for `expect` call?

Reply:
expect belongs to chai yes, but if you ask why there's that new chai version there I don't know. Yarn install is adding that automatically.